Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -1749,10 +1819,8 @@ def file_command(client: Client, score_calculator: ScoreCalculator, args: dict,
             # If anything happens, just keep going
             demisto.debug(f'Could not process file: "{file}"\n {str(exc)}')
             execution_metrics.general_error += 1
+            results.append(build_error_file_output(client, file))
             continue
-    if len(results) == 0:
```

Comment:
Then what do we do in this cases now?

Reply:
This should never happen because we are now appending an output in each branch, even when getting an exception.